src/tests/ftest/util/mdtest_utils.py

```diff
@@ -0,0 +1,216 @@
+    """Defines a object representing a mdtest command."""
+
+    def __init__(self):
+        """Create an MdtestCommand object."""
+        self.flags = MdtestParam("{}")                 # mdtest flags
+        self.api = MdtestParam("-a {}")                # API for I/O
+                                                       # [POSIX|DAOS|DFS]
+        self.branching_factor = MdtestParam("-b {}")   # branching factor
+        self.test_dir = MdtestParam("-d {}")           # test directory in
+                                                       # which tests should run
+        self.barriers = MdtestParam("-B {}")           # no barriers between
+                                                       # phases
+        self.read_bytes = MdtestParam("-e {}")         # bytes to read from
+                                                       # each file
+        self.first_num_tasks = MdtestParam("-f {}")    # first number of tasks
+                                                       # on which the test will
+                                                       # run
+        self.iteration = MdtestParam("-i {}")          # number of iterations
+        self.items = MdtestParam("-I {}")              # number of items per
+                                                       # directory in tree
+        self.last_num_tasks = MdtestParam("-l {}")     # last number of tasks
+        self.num_of_tasks = MdtestParam("-n {}")       # every process will
+                                                       # creat/stat/read/remove
+                                                       # num of  directories and
+                                                       # files
+        self.pre_iter = MdtestParam("-p {}")           # pre-iteration delay
+                                                       # (in seconds)
+        self.random_seed = MdtestParam("--random-seed {}") # random seed for -R
+                                                           # flag
+        self.stride = MdtestParam("-s {}")             # stride between the
+                                                       # number of tasks
+        self.verbosity_value = MdtestParam("-V {}")    # verbosity value
+        self.write_bytes = MdtestParam("-w {}")        # bytes to write to each
+                                                       # file
+        self.stonewall_timer = MdtestParam("-W {}")    # stonewall timer
+                                                       # (in secs)
+        self.stonewall_statusfile = MdtestParam("-x {}") # num of iter of the
+                                                       # creation phase,
+                                                       # StoneWallingStatusFile
+        self.depth = MdtestParam("-z {}")              # depth of hierarchical
+                                                       # directory structure
+
+        self.daos_pool_uuid = MdtestParam("--daos.pool {}") # pool uuid
```

Comment:
why do we set daos*** and then dfs*** ?
if this is just for mdtest, we don't need the --daos. options

Reply:
Will keep this for now as mdtest options show daos.* options.